content/source/docs/enterprise/api/admin/settings.html.md

```diff
@@ -0,0 +1,492 @@
+    }
+  }
+}
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request PATCH \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/admin/settings/saml
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id":"saml",
+    "type":"settings/saml",
+    "attributes": {
+      "enabled": true,
+      "idp-cert": "SAMPLE-CERTIFICATE",
+      "slo-target": "https://example.com/slo",
+      "sso-target": "https://example.com/sso",
+      "attr-groups": "MemberOf",
+      "attr-site-admin": "SiteAdmin",
+      "site-admin-role": "site-admins",
+      "sso-api-token-session-timeout": 1209600
+    }
+  }
+}
+```
+
+## List SMTP Settings
+`GET /api/v2/admin/settings/smtp`
+
+-> **Note:** This endpoint cannot be accessed with [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) which has the `site-admins` permission.
+
+Status  | Response                                         | Reason
+--------|--------------------------------------------------|-------
+[200][] | [JSON API document][] (`type: "settings/saml"`) | Successfully listed SMTP settings
+[404][] | [JSON API error object][]                    | User unauthorized to perform action
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[JSON API document]: https://www.terraform.io/docs/enterprise/api/index.html#json-api-documents
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request GET \
+  https://app.terraform.io/api/v2/admin/settings/smtp
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "smtp",
+    "type": "settings/smtp",
+    "attributes": {
+      "enabled": true,
+      "host": "example.com",
+      "port": 25,
+      "sender": "sample_user@example.com",
+      "auth": "login",
+      "username": "sample_user"
+    }
+  }
+}
+```
+
+## Update SMTP Settings
+`PATCH /api/v2/admin/settings/smtp`
+
+-> **Note:** This endpoint cannot be accessed with [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) which has the `site-admins` permission.
+
+Status  | Response                                     | Reason
+--------|----------------------------------------------|----------
+[200][] | [JSON API document][] (`type: "settings/smtp"`) | Successfully updated the SMTP settings
+[401][] | [JSON API error object][]                    | Failure during test message delivery, SMTP user credentials are invalid
```

Comment:
The "failure during test message delivery" here confuses me. Are we validating the creds by sending a test message? If so, that should be included in the description of the endpoint, and then I think the error here could be abbreviated to `SMTP user credentials are invalid` and likewise for the below codes that mention the test message.

Something like:

"When this request is submitted, (what happens with the test message here). If the test message delivery fails, the API will return an error code indicating the reason for the failure."